types/audit.d.ts

```diff
@@ -57,6 +57,8 @@ declare global {
       __internalOptionalArtifacts?: Array<keyof Artifacts>;
       /** A string identifying how the score should be interpreted for display. */
       scoreDisplayMode?: Audit.ScoreDisplayMode;
+      /** A list of gather modes that this audit is applicable to. */
+      applicableModes?: Gatherer.GatherMode[],
```

Comment:
WDYT about matching `supportedModes`? was the use of `applicable` instead an intentional differentiator?

Reply:
It was different intentionally to avoid mismatch with gatherers. `supportedModes` does make sense though so I'm fine going back.